stubs/model-factory.stub

```diff
@@ -6,7 +6,7 @@ use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Factories\Factory;
 
-use {{ factory_contract_namespace }}\{{ factory_contract }};
+use {{ factoryContractNamespace }}{{ factoryContract }}
```

Comment:
last time @holiq change the namespace using backslash between contract name and factorycontract is it ady covered?